cmd/root.go

```diff
@@ -1,20 +1,22 @@
+// Package cmd implements fleetlockctl CLI.
 package cmd
 
 import (
 	"github.com/spf13/cobra"
 )
 
-var (
-	group, id, url string
+// Command returns CLI command to be executed.
+func Command() *cobra.Command {
+	cli := &cobra.Command{Use: "fleetlockctl"}
 
-	Root = &cobra.Command{Use: "fleetlockctl"}
-)
+	var group, id, url string
+
+	cli.PersistentFlags().StringVarP(&group, "group", "g", "default", "FleetLock group")
+	cli.PersistentFlags().StringVarP(&id, "id", "i", "", "FleetLock instance ID (/etc/machine-id for example)")
```

Comment:
While you're here:
```suggestion
	cli.PersistentFlags().StringVarP(&id, "id", "i", "", "FleetLock instance ID (content of /etc/machine-id for example)")
```

Reply:
Yeah, I was thinking about this thing :smile: I was unsure about FleetLock prefix for all flags as well :smile:

Reply:
I'll do `FleetLock instance ID (e.g. content of /etc/machine-id file)` to keep the line below 120 characters.

Reply:
Fixed.